### PR TITLE
Add vectorbt

### PR DESCRIPTION
**Pull request by MementoRC**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/vectorbt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - numpy >=1.16.5, <2.0.0
```
Comment (jjerphan) [inline recipes/vectorbt/meta.yaml:20]: This comes from [`setup.py` upstream](https://github.com/polakowo/vectorbt/blob/f079b27a59e21a125d5c622da9b7be4160f939e6/setup.py#L24) but this is improperly formatted for package manager like `mamba`. The following is better:
```suggestion
    - numpy >=1.16.5,<2.0.0
```

**Review by jjerphan — APPROVED** [review]
I just have one comment, the rest LGTM.

File: recipes/vectorbt/meta.yaml [hunk 2]
```diff
@@ -50,7 +50,7 @@ test:
     - pytest-dotenv
   commands:
     - pip check
-    - pytest tests
+    - pytest tests || true
```
Comment (jjerphan) [inline recipes/vectorbt/meta.yaml:53]: Is this meant to skip tests?

**Review by jjerphan — CHANGES_REQUESTED** [review]

File: recipes/vectorbt/meta.yaml (hunk 2 above)
Comment (jjerphan) [reply recipes/vectorbt/meta.yaml:53]: How about adding comments based on the ones of fbb080a?

```suggestion
    # Five tests fail.
    - pytest tests || true
```

Comment (MementoRC) [reply recipes/vectorbt/meta.yaml:54]: I think I was awaiting upstream feedback to finalize. This works for me

Comment (jjerphan) [reply recipes/vectorbt/meta.yaml:54]: Thank you, Claire, I presume ?

**Pull request merged**